### PR TITLE
feat(reportingApp): document the Bonita enterprise for reporting application 

### PR DESCRIPTION
**Pull request by vbotnari12**
Linked to Runtime-1280

### CODE REVIEW

File: modules/runtime/pages/reporting-app.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+An Enterprise licence is required. 
+====
+
+== Overview 
+
+The bonita-dashboard-app-1.1.1.jar, contains two reports “Process Historical Data” and “Human tasks historical data”.  
+
+=== Process historical data
+
+For analyzing zases execution data of a chosen process in a defined period of time.
```
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: Typo: cases

File: modules/runtime/pages/reporting-app.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+On the analysis page you’ll have the possibility to create the graph with cases average time and/or download the file with detailed archived cases data in .csv format inside a zip. This enables the data to be sorted and filtered.
```
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: What enables the sorting and filtering? ==> I think the 2 sentences should be modified to express a single idea each time. E.g:
From the analysis page you'll have the option to consume the data either by displaying a graph or by downloading them.
The Graph shows....
The downloaded data is packaged as .... and allows you to analyse them in Excel or your BI tool of your choice...

File: modules/runtime/pages/reporting-app.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+=== Human tasks historical data
+
+For analyzing human actions in cases of a chosen process in a defined period of time. 
+
+On the analysis page you’ll have the possibility to create the graph with tasks average handling time and/or download the file with detailed archived human tasks data in .csv format inside a zip. This enables the data to be sorted and filtered.
```
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: same comment as above

File: modules/runtime/pages/reporting-app.adoc [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+== Diferentiators
```
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: "Differentiators" compared to what? 
Why is this section relevant to the reader who wants to understand the usage of this application?

File: modules/runtime/pages/reporting-app.adoc (hunk 1 above)
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: What are the main use cases of this page / report? What can the user do / understand by reading this report?

File: modules/runtime/pages/reporting-app.adoc [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+=== Process historical data
+
+For analyzing zases execution data of a chosen process in a defined period of time.
+
+On the analysis page you’ll have the possibility to create the graph with cases average time and/or download the file with detailed archived cases data in .csv format inside a zip. This enables the data to be sorted and filtered.
+
+=== Human tasks historical data
+
+For analyzing human actions in cases of a chosen process in a defined period of time.
```
Comment (nchabanoles) [inline modules/runtime/pages/reporting-app.adoc]: What are the main use cases of this page / report? What can the user do / understand by reading this report?

File: modules/runtime/pages/reporting-app.adoc (hunk 1 above)
Comment (nchabanoles) [reply modules/runtime/pages/reporting-app.adoc]: More over a screenshot of the graph and the explanations on the 2 axis could be very instructive to understand what the graph shows, what do you think?

Comment (vbotnari12) [reply modules/runtime/pages/reporting-app.adoc]: changed.

File: modules/runtime/pages/reporting-app.adoc (hunk 5 above)
Comment (vbotnari12) [reply modules/runtime/pages/reporting-app.adoc]: it's in the description "Bonita Reporting app will enable you to view statistics and to extract process execution reports with meaningful data from Bonita, for process performance analysis and for pinpointing areas of improvement.", I think...

File: modules/runtime/pages/reporting-app.adoc (hunk 1 above)
Comment (vbotnari12) [reply modules/runtime/pages/reporting-app.adoc]: it's in the description "Bonita Reporting app will enable you to view statistics and to extract process execution reports with meaningful data from Bonita, for process performance analysis and for pinpointing areas of improvement.", I think...
Picture added :)

File: modules/runtime/pages/reporting-app.adoc (hunk 3 above)
Comment (vbotnari12) [reply modules/runtime/pages/reporting-app.adoc]: Adapted.

File: modules/runtime/pages/reporting-app.adoc (hunk 2 above)
Comment (vbotnari12) [reply modules/runtime/pages/reporting-app.adoc]: adapted and added the graph.

**Pull request merged**